_layouts/page.html

```diff
@@ -64,7 +64,7 @@
         {% if page.title %}
           {% unless page.title_hidden %}
             <div class="page-header">
-              <h1>{{ page.title }}</h1>
```

Comment:
Please look for pages, that have a custom page title (e.g. `title_hidden: true` in their frontmatter) and look, if the custom page header there also needs the `class=title` attribute.